content/blog/cloud-native-infrastructure-with-kubernetes-and-pulumi/index.md

```diff
@@ -7,28 +7,26 @@ date: "2018-09-12"
 meta_image: "diff.gif"
 ---
 
-Today we're pleased to
-[announce Pulumi for Kubernetes](https://info.pulumi.com/press-release/pulumi-cloud-native-sdk-delivers-cloud-native-infrastructure-as-code-for-kubernetes),
+Kubernetes has quickly become the "gold standard" for running containers
+in production, spanning public, private, and hybrid cloud scenarios.
+It's been remarkable to watch its explosive progress in even just this
+past year alone. Every cloud vendor now supports an easy-to-use managed
+Kubernetes solution -- Google GKE, Azure AKS, and AWS AKS -- making it
```

Comment:
The markdown renderer will automatically convert `--` to `&ndash;`, and `---` to `&mdash;` via "LaTeX-style dash parsing". I wonder if we disable that extension if `-` would be converted to `&ndash;` and `--` would be converted to `&mdash;`.

Reply:
I updated my suggestion to use `---` instead. If this isn't in CONTRIBUTING.md, I'll add it since this is better than having writers figure out HTML entities and symbols.